build/yaml/crd/legacy/nsx.vmware.com_securitypolicies.yaml

```diff
@@ -448,7 +448,6 @@ spec:
                             description: Port is the name or port number.
                             x-kubernetes-int-or-string: true
                           protocol:
-                            default: TCP
```

Comment:
Thanks @yuntanghsu,
For this T1 CRD, 
could you help have a try to change CRD definition:
https://github.com/vmware-tanzu/nsx-operator/blob/main/pkg/apis/legacy/v1alpha1/securitypolicy_types.go#L103
 
to add one kubebuilder directive:
`// +kubebuilder:default=TCP`

to see if the controller-gen can generate default value for protocol field.

Reply:
Thanks @timdengyun , I've added back. Please take a look again. Thanks!

Reply:
Hi @yuntanghsu I see you changed VPC securitypolicy_types.go, it looks like working. 
Could you also help to change legacy T1 type:
https://github.com/vmware-tanzu/nsx-operator/blob/main/pkg/apis/legacy/v1alpha1/securitypolicy_types.go#L103
and re generated CRD yaml for T1. Thanks